src/model_signing/_cli.py

```diff
@@ -443,7 +469,7 @@ def hasher_factory(file_path: pathlib.Path) -> file.FileHasher:
             verifier=verifier,
             model_path=model_path,
             serializer=serializer,
-            ignore_paths=[signature_file],
+            ignore_paths=[signature_file] + ignore_paths,
```

Comment:
Let's write this as `ignore_paths + [signature_file]` (or extract to a separate list before). It should be slightly faster

```python
>>> import timeit
>>> timeit.repeat("one+many", setup="one=[1]; many=list(range(1000))")
[1.4256296629901044, 1.4093451339867897, 1.427245083003072, 1.357308656006353, 1.3508969740069006]
>>> timeit.repeat("many+one", setup="one=[1]; many=list(range(1000))")
[1.381321033986751, 1.3717010380059946, 1.3515870239934884, 1.3433384770178236, 1.3438171989982948]
```